sql-plugin/src/main/scala/com/nvidia/spark/rapids/RegexParser.scala

```diff
@@ -176,6 +176,8 @@ class RegexParser(pattern: String) {
                 case Some(end) =>
                   skip()
                   characterClass.appendRange(start, end)
+                case _ =>
+                  // TODO @andygrove
```

Comment:
This should throw an exception since that's what it did before.  If it's not supposed to throw then that can be fixed in a followup specific to the regex parser.